smtk/io/XmlDocV1Parser.cxx

```diff
@@ -2357,7 +2357,8 @@ void XmlDocV1Parser::processViews(xml_node &root)
     {
     return;
     }
-  smtk::common::ViewPtr rootView = this->createView(views, "Root");
+  smtk::common::ViewPtr rootView = this->createView(views, "Group");
+  rootView->details().setAttribute("TopLevel", "true");
```

Comment:
Latest push addresses both Yumin's and John/s comments.

Bob

Robert M. O'Bara, MEng.
Assistant Director of Scientific Computing

Kitware Inc.
28 Corporate Drive
Suite 101
Clifton Park, NY 12065

Phone: (518) 881- 4931




> On Sep 18, 2015, at 1:53 PM, Yumin Yuan <notifications@github.com> wrote:
> 
> In smtk/io/XmlDocV1Parser.cxx <https://github.com/Kitware/SMTK/pull/270#discussion_r39882214>:
> 
> > @@ -2357,7 +2357,8 @@ void XmlDocV1Parser::processViews(xml_node &root)
> >      {
> >      return;
> >      }
> > -  smtk::common::ViewPtr rootView = this->createView(views, "Root");
> > +  smtk::common::ViewPtr rootView = this->createView(views, "Group");
> > +  rootView->details().setAttribute("TopLevel", "true");
> This line of code is duplicated below
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/Kitware/SMTK/pull/270/files#r39882214>.
>